### PR TITLE
Add PHPStan & Pint

### PR DESCRIPTION
**Pull request by Jade-GG**
Also ran them and made any relevant fixes :)

### CODE REVIEW

**Comment by royduin** [comment]
Can we keep the indenting? Config files are better readable in the old situation

**Comment by Jade-GG** [comment]
> Can we keep the indenting? Config files are better readable in the old situation

Fix'd!

File: config/debugbar.php
Comment (royduin) [inline config/debugbar.php:1]: Most likely this file is pretty outdated and can be removed

**Comment by royduin** [comment]
Isn't https://github.com/rapidez/rapidez/blob/master/.github/workflows/copy-laravel-source.yml overwriting anything back to what it was?

**Comment by Jade-GG** [comment]
> Isn't https://github.com/rapidez/rapidez/blob/master/.github/workflows/copy-laravel-source.yml overwriting anything back to what it was?

Hmm, that's annoying, because phpstan will complain about the unnecessary imports. I could just ignore that error instead...

File: pint.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "preset": "laravel",
+    "rules": {
+        "binary_operator_spaces": false,
+        "no_unused_imports": false
```
Comment (royduin) [inline pint.json:5]:
```suggestion
        "binary_operator_spaces": false
```

**Pull request merged**